src/Cogworks.CogFlare.Core/Services/CachePurgeService.cs

```diff
@@ -33,13 +33,15 @@ public async Task PurgeExternalCacheAsync(
         string notificationLabel,
         bool isMedia = false)
     {
-        var urlsToPurge = new List<string>();
-
         foreach (var id in ids)
         {
             if (IsKeyNode(id))
             {
-                _logger.LogInformation($"Full purge triggered: [{id}] Key node {notificationLabel}");
+                if (_cogFlareSettings.EnableLogging)
```

Comment:
Can we extract this into a method and then only pass the string to it? So that there will be 1 if statement checking the flag rather than multiple.